lib/hello_world_web/controllers/api/message_controller.ex

```diff
@@ -0,0 +1,21 @@
+defmodule HelloWorldWeb.API.MessageController do
+  use HelloWorldWeb, :controller
+
+  @spec public(Plug.Conn.t(), any) :: Plug.Conn.t()
+  def public(conn, _params) do
+    message = "The API doesn't require an access token to share this message."
```

Comment:
@manusajith maybe `message` should be defined as a constant?

Reply:
We could use [module attributes](https://elixir-lang.org/getting-started/module-attributes.html#as-constants), but messages are not re-used between actions. So having them in the scope of the action itself might be better in this case.